mmdet/models/necks/hrfpn.py

```diff
@@ -0,0 +1,87 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from torch.utils.checkpoint import checkpoint
+
+from ..registry import NECKS
+
+
+@NECKS.register_module
+class HRFPN(nn.Module):
+
+    def __init__(self,
+                 in_channels,
+                 out_channels,
+                 normalize=None,
+                 pooling='AVG',
+                 share_conv=False,
+                 with_cp=False):
+        super(HRFPN, self).__init__()
+        assert isinstance(in_channels, list)
+        self.in_channels = in_channels
+        self.out_channels = out_channels
+        self.num_ins = len(in_channels)
+        self.with_bias = normalize is None
+        self.share_conv = share_conv
+        self.reduction_conv = nn.Sequential(
+            nn.Conv2d(in_channels=sum(in_channels),
+                      out_channels=out_channels,
+                      kernel_size=1),
+        )
+
+        if self.share_conv:
+            self.fpn_conv = nn.Conv2d(in_channels=out_channels,
+                                      out_channels=out_channels,
+                                      kernel_size=3, padding=1)
+        else:
+            self.fpn_conv = nn.ModuleList()
+            for i in range(5):
```

Comment:
Is it possible to remove this hardcoded number? e.g., to support arbitrary number of levels.